### PR TITLE
update Readme

### PR DESCRIPTION
**Pull request by SimonLab**
ref: #6 

- Update README before publishing package
- Add LICENSE file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,86 @@
...
+view model =
+    div []
+        [
+          Criteria.view criteriaConfig model.criteria filters
+        ]
+```
 ## Examples
 
-[example](https://dwyl.github.io/elm-criteria/example.html)
+See a [live example](https://dwyl.github.io/elm-criteria/example.html) and its [code]()
```
Comment (nelsonic) [inline README.md:69]: Remember to update the direct link to the code for the example. 👍🏻

**Review by nelsonic — APPROVED** [review]
@SimonLab looks good.
We don’t “normally” use BSD license, but I have no “objection”.
Happy to merge.
Thanks! 🙌

**Comment by SimonLab** [comment]
Thanks for the review/merge @nelsonic 
I wasn't sure which license to use but saw that BSD was mostly used with Elm package and was recommended. We can review it later if necessary :+1:

**Comment by nelsonic** [comment]
@SimonLab yeah, BSD is a _good_ license. 
We tend to use GPL-2 see: https://github.com/dwyl/intellectual-property
Elm package is recommending a more "permissive" license. No need to change. 
Thanks for moving fast with this project. ✨

**Pull request merged**